chapter4.adoc

```diff
@@ -221,9 +220,54 @@ follows:
 ], config:{lanes: 1, hspace:1024}}
 ....
 
+=== MTT access permissions lookup process
+
+MTT access permissions for a physical address PA in the context of a
+supervisor domain is ascertained as follows:
+
+1. Let a be `mttp.ppn`x`PAGESIZE`, and let i = LEVELS, where for mode
+Smmtt[34][*], LEVELS = 2 and for Smmtt[46, 56][*rw*] LEVELS = 3; PAGESIZE is
```

Comment:
Smmtt[34][*] -> Smmtt34*
Smmtt[46, 56][*rw*] -> Smmtt[46|56]*

Reply:
I thought LEVELS=3 only applies to Smmtt56.

Reply:
for 46 and 56 the level should be 3.